lib/echo_server.ex

```diff
@@ -1,18 +1,25 @@
 defmodule EchoServer do
-  @moduledoc """
-  Documentation for `EchoServer`.
-  """
+  use GenServer
 
-  @doc """
-  Hello world.
+  # Client
 
-  ## Examples
+  def start_link() do
+    GenServer.start_link(__MODULE__, [])
+  end
+
+  def send_message(pid, :ping) do
+    GenServer.call(pid,  {:send_message, :ping})
```

Comment:
```suggestion
    GenServer.call(pid,  :ping)
```
Избыточность нам без нужды :)